openhands/sdk/__init__.py

```diff
@@ -57,5 +65,8 @@
     "LLMSummarizingCondenser",
     "FileStore",
     "LocalFileStore",
+    "register_tool",
+    "resolve_tool",
```

Comment:
```suggestion
```

NIT: do we need this in the init class?

Reply:
I guess it is handy to keep it there -- Agent need it to convert spec back to tool instances